src/components/typeahead/typeahead.component.ts

```diff
@@ -255,6 +255,7 @@ export class TypeaheadComponent<T = any> implements OnChanges, OnDestroy {
     ngOnDestroy(): void {
         this._onDestroy.next();
         this._onDestroy.complete();
+        this._popoverOrientation.createPopoverOrientationListener(this.typeaheadElement.nativeElement, this.typeaheadElement.nativeElement.parentElement).destroy();

```

Comment:
You'll need to save the original instance as a private property and destroy that, this is just creating a second set of resources and cleaning those up.